julia_src/reopt_model.jl

```diff
@@ -195,15 +223,15 @@ function add_bigM_adjustments(m, p)
 	for n in p.DemandMonthsBin
 		for mth in p.Month
 			if n > 1
-				m[:NewMaxDemandMonthsInTier][mth,n] = minimum([p.MaxDemandMonthsInTier[n],
-					added_power + 2*maximum([p.ElecLoad[ts] + p.CoolingLoad[ts] / p.ElectricChillerCOP
-					for ts in p.TimeStepRatchetsMonth[mth]])  -
-					sum(m[:NewMaxDemandMonthsInTier][mth,np] for np in 1:(n-1)) ]
-				)
+				m[:NewMaxDemandMonthsInTier][mth,n] = minimum([p.MaxDemandMonthsInTier[n], DemandBigM])
+				# 	added_power + 2*maximum([p.ElecLoad[ts] + p.CoolingLoad[ts] / p.ElectricChillerCOP
```

Comment:
there are a lot of commented out lines in `add_bigM_adjustments`: why is that?

Reply:
The commented-out bigM heuristics don't work for GHP because we are electrifying the heating load, so the electric load demand/usage bigM can't be based on the BAU electric load alone. I've effectively replaced the heuristic NewMaxDemand/Usage with a big number (if there are not tiers, otherwise use the tiered parameters).

Reply:
I just uncommented out the heuristic bigM calcs and added GHP-specific adjustments to the calculations, so this preserves the previous logic. I've added the heating load to the electric-based bigM calcs if GHP is considered (implicitly assumes a worst-case COP of 1 for GHP), and I removed the electric chiller COP from the CoolingLoad term in case the GHP COP is lower than the BAU electric chiller COP.